doc: refactoring (#1253)
AntaresSimulatorTeam/antares-xpansion: docs/reference/outputs/logs.md

> # Logs

Reviewer: You should also mention the `benders_output_trace.csv` (you can look at the source code to understand all information gathered in it or use a LLM to retrieve that quickly)

Other files are produced but that are not logs :
- master_last_iteration (.svf or .mps depending on the solver) to store the cuts at the end of a Benders run
- master_last_basis.bss (only with Xpress ?) : Basis of the last master
Both files are useful (used as inputs) for the sensitivty analysis

Several log files are written:

- `reportbenders.txt` gives information on the progress of the algorithm with an operational perspective,

> - `benders_solver.log` contains more detailed information on all data of interest to follow the progress of the algorithm (`lambda_min`, `lambda_max`, detailed solving times, ...).

Reviewer: ```suggestion
- `benders_solver.log` is the mathematical log of the Benders solver. It contains more detailed information on all data of interest to follow the progress of the algorithm (`lambda_min`, `lambda_max`, detailed solving times, ...).
```

> - The file `LOLD.txt` under `lp/` dir stores adequacy criteria for all valid patterns (area+criterion)

Reviewer: ```suggestion
- The file `LOLD.txt` under `lp/` dir stores the number of hours of loss of load for each area at each iteration of the algorithm.
```

> - The file `UnsuppliedEnergy.txt` under `lp/` dir stores the amount of unsupplied energy for all valid patterns (area+criterion)

Reviewer: ```suggestion
- The file `UnsuppliedEnergy.txt` under `lp/` dir stores the amount of unsupplied energy for each area at each iteration of the algorithm.
```
